INFO 2025-10-18 11:44:01,216 autoreload 18128 1652 Watching for file changes with StatReloader
INFO 2025-10-18 11:45:04,280 autoreload 18128 1652 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\pristineprimer\settings.py changed, reloading.
INFO 2025-10-18 11:45:05,970 autoreload 24264 22800 Watching for file changes with StatReloader
ERROR 2025-10-18 11:46:05,048 log 24264 18152 Internal Server Error: /
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\loader.py", line 47, in select_template
    raise TemplateDoesNotExist(", ".join(template_name_list), chain=chain)
django.template.exceptions.TemplateDoesNotExist: index.html
ERROR 2025-10-18 11:46:05,055 basehttp 24264 18152 "GET / HTTP/1.1" 500 91662
INFO 2025-10-18 11:46:18,083 basehttp 24264 20432 "GET /admin/ HTTP/1.1" 302 0
INFO 2025-10-18 11:46:18,131 basehttp 24264 20432 "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4205
INFO 2025-10-18 11:46:18,348 basehttp 24264 20432 "GET /static/admin/css/login.css HTTP/1.1" 200 951
WARNING 2025-10-18 11:46:18,970 log 24264 20432 Not Found: /api/orders/
WARNING 2025-10-18 11:46:18,972 basehttp 24264 20432 "GET /api/orders/ HTTP/1.1" 404 11022
INFO 2025-10-18 11:52:04,908 basehttp 24264 2696 "GET /admin/payments/mpesaconfiguration/add/ HTTP/1.1" 302 0
INFO 2025-10-18 11:52:04,967 basehttp 24264 2696 "GET /admin/login/?next=/admin/payments/mpesaconfiguration/add/ HTTP/1.1" 200 4269
INFO 2025-10-18 12:48:36,285 basehttp 24264 23192 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 12:48:36,600 basehttp 24264 23916 "OPTIONS /api/auth/me/ HTTP/1.1" 200 0
INFO 2025-10-18 12:48:36,619 basehttp 24264 23192 "GET /api/auth/me/ HTTP/1.1" 200 32
INFO 2025-10-18 12:48:40,274 basehttp 24264 23192 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 12:48:40,282 basehttp 24264 23192 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 12:48:40,516 basehttp 24264 23916 "OPTIONS /api/properties/?featured=true&limit=6 HTTP/1.1" 200 0
INFO 2025-10-18 12:48:40,621 basehttp 24264 2100 "GET /api/auth/me/ HTTP/1.1" 200 32
INFO 2025-10-18 12:48:41,272 basehttp 24264 23192 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 788
INFO 2025-10-18 12:48:41,397 basehttp 24264 23192 "GET /media/property_images/ChatGPT_Image_Sep_13_2025_02_47_02_PM_obfB0IH.png HTTP/1.1" 200 2462478
INFO 2025-10-18 12:48:41,460 basehttp 24264 23192 "GET /media/property_images/ChatGPT_Image_Sep_13_2025_02_47_02_PM.png HTTP/1.1" 200 2462478
INFO 2025-10-18 13:04:12,387 basehttp 24264 20328 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 13:04:12,424 basehttp 24264 20328 "GET /api/auth/me/ HTTP/1.1" 200 32
INFO 2025-10-18 13:18:08,054 basehttp 24264 19668 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 13:18:08,061 basehttp 24264 19668 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 13:18:08,290 basehttp 24264 19668 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 788
INFO 2025-10-18 13:18:08,295 basehttp 24264 19668 "GET /api/auth/me/ HTTP/1.1" 200 32
INFO 2025-10-18 13:25:31,384 basehttp 24264 21432 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 13:25:31,613 basehttp 24264 21432 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 788
INFO 2025-10-18 13:27:22,077 basehttp 24264 21316 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 13:27:22,250 basehttp 24264 21316 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 788
INFO 2025-10-18 13:27:34,245 basehttp 24264 20012 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 13:27:34,572 basehttp 24264 14628 "OPTIONS /api/properties/2/ HTTP/1.1" 200 0
INFO 2025-10-18 13:27:34,721 basehttp 24264 20012 "GET /api/properties/2/ HTTP/1.1" 200 837
INFO 2025-10-18 13:27:45,897 basehttp 24264 5184 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 13:27:46,071 basehttp 24264 5184 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 788
INFO 2025-10-18 13:27:53,459 basehttp 24264 10116 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 13:27:53,798 basehttp 24264 24248 "OPTIONS /api/properties/1/ HTTP/1.1" 200 0
INFO 2025-10-18 13:27:53,947 basehttp 24264 10116 "GET /api/properties/1/ HTTP/1.1" 200 1003
INFO 2025-10-18 13:27:53,998 basehttp 24264 10116 "GET /media/property_images/delivery-man-on-motorcycle-with-package-vector.jpg HTTP/1.1" 200 72060
INFO 2025-10-18 13:37:21,426 basehttp 24264 18092 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 13:37:21,610 basehttp 24264 18092 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 788
INFO 2025-10-18 14:35:23,755 autoreload 24264 22800 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\properties\models.py changed, reloading.
INFO 2025-10-18 14:35:27,107 autoreload 25176 23704 Watching for file changes with StatReloader
INFO 2025-10-18 14:39:55,688 autoreload 25176 23704 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\properties\serializers.py changed, reloading.
INFO 2025-10-18 14:39:57,183 autoreload 4952 3076 Watching for file changes with StatReloader
INFO 2025-10-18 14:40:04,671 autoreload 4952 3076 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\properties\serializers.py changed, reloading.
INFO 2025-10-18 14:40:06,298 autoreload 8216 16460 Watching for file changes with StatReloader
INFO 2025-10-18 14:40:17,651 autoreload 8216 16460 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\properties\serializers.py changed, reloading.
INFO 2025-10-18 14:40:18,763 autoreload 1500 21624 Watching for file changes with StatReloader
INFO 2025-10-18 14:40:32,780 autoreload 1500 21624 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\properties\serializers.py changed, reloading.
INFO 2025-10-18 14:40:33,983 autoreload 3480 12760 Watching for file changes with StatReloader
INFO 2025-10-18 15:02:59,949 autoreload 3480 12760 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\properties\admin.py changed, reloading.
INFO 2025-10-18 15:03:02,016 autoreload 16224 22200 Watching for file changes with StatReloader
INFO 2025-10-18 15:08:07,977 autoreload 16224 22200 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\properties\views.py changed, reloading.
INFO 2025-10-18 15:08:09,773 autoreload 8860 16236 Watching for file changes with StatReloader
INFO 2025-10-18 15:11:06,325 autoreload 8860 16236 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\properties\serializers.py changed, reloading.
INFO 2025-10-18 15:11:08,247 autoreload 15632 19620 Watching for file changes with StatReloader
INFO 2025-10-18 15:16:58,564 autoreload 15632 19620 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\properties\filters.py changed, reloading.
INFO 2025-10-18 15:17:00,248 autoreload 19664 21996 Watching for file changes with StatReloader
INFO 2025-10-18 15:17:05,633 autoreload 19664 21996 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\properties\filters.py changed, reloading.
INFO 2025-10-18 15:17:07,979 autoreload 21304 24944 Watching for file changes with StatReloader
INFO 2025-10-18 15:17:10,535 autoreload 21304 24944 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\properties\filters.py changed, reloading.
INFO 2025-10-18 15:17:11,965 autoreload 19936 2160 Watching for file changes with StatReloader
INFO 2025-10-18 15:18:57,200 autoreload 19936 2160 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\pristineprimer\settings.py changed, reloading.
INFO 2025-10-18 15:18:58,685 autoreload 17276 13204 Watching for file changes with StatReloader
INFO 2025-10-18 15:22:16,842 autoreload 17276 13204 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\properties\urls.py changed, reloading.
INFO 2025-10-18 15:22:18,587 autoreload 24648 21448 Watching for file changes with StatReloader
INFO 2025-10-18 15:22:30,095 autoreload 24648 21448 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\properties\urls.py changed, reloading.
INFO 2025-10-18 15:22:31,580 autoreload 25008 4420 Watching for file changes with StatReloader
INFO 2025-10-18 15:26:13,445 autoreload 25008 4420 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\properties\models.py changed, reloading.
INFO 2025-10-18 15:26:15,638 autoreload 8800 3312 Watching for file changes with StatReloader
INFO 2025-10-18 15:26:20,798 autoreload 8800 3312 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\properties\models.py changed, reloading.
INFO 2025-10-18 15:26:22,064 autoreload 23952 2076 Watching for file changes with StatReloader
INFO 2025-10-18 15:27:42,470 autoreload 23952 2076 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\properties\admin.py changed, reloading.
INFO 2025-10-18 15:27:43,570 autoreload 20280 21952 Watching for file changes with StatReloader
INFO 2025-10-18 15:30:17,819 autoreload 20280 21952 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\properties\urls.py changed, reloading.
INFO 2025-10-18 15:30:19,898 autoreload 11696 2056 Watching for file changes with StatReloader
INFO 2025-10-18 15:32:08,285 autoreload 11696 2056 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\properties\urls.py changed, reloading.
INFO 2025-10-18 15:32:10,615 autoreload 14700 10060 Watching for file changes with StatReloader
INFO 2025-10-18 15:36:01,815 autoreload 14700 10060 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\properties\urls.py changed, reloading.
INFO 2025-10-18 15:36:03,916 autoreload 17520 25060 Watching for file changes with StatReloader
INFO 2025-10-18 15:36:46,659 autoreload 17520 25060 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\properties\urls.py changed, reloading.
INFO 2025-10-18 15:36:48,132 autoreload 13280 9436 Watching for file changes with StatReloader
INFO 2025-10-18 15:37:22,448 basehttp 13280 13644 "GET /admin/ HTTP/1.1" 302 0
INFO 2025-10-18 15:37:22,555 basehttp 13280 13644 "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4205
INFO 2025-10-18 15:37:22,848 basehttp 13280 13964 "GET /static/admin/js/theme.js HTTP/1.1" 200 1653
INFO 2025-10-18 15:37:22,853 basehttp 13280 24300 "GET /static/admin/css/base.css HTTP/1.1" 200 22120
INFO 2025-10-18 15:37:22,854 basehttp 13280 13644 "GET /static/admin/css/dark_mode.css HTTP/1.1" 200 2808
INFO 2025-10-18 15:37:22,855 basehttp 13280 13964 "GET /static/admin/css/nav_sidebar.css HTTP/1.1" 200 2810
INFO 2025-10-18 15:37:22,868 basehttp 13280 13964 "GET /static/admin/css/responsive.css HTTP/1.1" 200 16565
INFO 2025-10-18 15:37:22,871 basehttp 13280 13644 "GET /static/admin/js/nav_sidebar.js HTTP/1.1" 200 3063
ERROR 2025-10-18 15:37:23,894 log 13280 13644 Internal Server Error: /favicon.ico
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\loader.py", line 47, in select_template
    raise TemplateDoesNotExist(", ".join(template_name_list), chain=chain)
django.template.exceptions.TemplateDoesNotExist: index.html
ERROR 2025-10-18 15:37:23,906 basehttp 13280 13644 "GET /favicon.ico HTTP/1.1" 500 92199
INFO 2025-10-18 15:37:34,554 basehttp 13280 19528 "POST /admin/login/?next=/admin/ HTTP/1.1" 302 0
INFO 2025-10-18 15:37:34,659 basehttp 13280 19528 "GET /admin/ HTTP/1.1" 200 20863
INFO 2025-10-18 15:37:34,719 basehttp 13280 19528 "GET /static/admin/css/dashboard.css HTTP/1.1" 200 441
INFO 2025-10-18 15:37:34,739 basehttp 13280 19528 "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 200 331
INFO 2025-10-18 15:37:34,739 basehttp 13280 23604 "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 200 380
ERROR 2025-10-18 15:37:35,029 log 13280 23604 Internal Server Error: /favicon.ico
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\loader.py", line 47, in select_template
    raise TemplateDoesNotExist(", ".join(template_name_list), chain=chain)
django.template.exceptions.TemplateDoesNotExist: index.html
ERROR 2025-10-18 15:37:35,036 basehttp 13280 23604 "GET /favicon.ico HTTP/1.1" 500 92343
INFO 2025-10-18 15:37:49,901 basehttp 13280 2636 "GET /admin/properties/property/add/ HTTP/1.1" 200 101741
INFO 2025-10-18 15:37:49,954 basehttp 13280 22228 "GET /static/admin/css/autocomplete.css HTTP/1.1" 200 9185
INFO 2025-10-18 15:37:49,955 basehttp 13280 2636 "GET /static/admin/css/vendor/select2/select2.css HTTP/1.1" 200 17358
INFO 2025-10-18 15:37:49,969 basehttp 13280 2636 "GET /static/admin/js/autocomplete.js HTTP/1.1" 200 1060
INFO 2025-10-18 15:37:49,974 basehttp 13280 22228 "GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 200 285314
INFO 2025-10-18 15:37:49,978 basehttp 13280 2636 "GET /static/admin/js/jquery.init.js HTTP/1.1" 200 347
INFO 2025-10-18 15:37:49,978 basehttp 13280 1228 "GET /static/admin/js/inlines.js HTTP/1.1" 200 15628
INFO 2025-10-18 15:37:49,983 basehttp 13280 23816 "GET /static/admin/js/vendor/select2/select2.full.js HTTP/1.1" 200 173566
INFO 2025-10-18 15:37:49,994 basehttp 13280 2636 "GET /static/admin/js/actions.js HTTP/1.1" 200 8076
INFO 2025-10-18 15:37:49,984 basehttp 13280 21952 "GET /static/admin/css/forms.css HTTP/1.1" 200 8525
INFO 2025-10-18 15:37:49,987 basehttp 13280 22228 "GET /static/admin/js/core.js HTTP/1.1" 200 6208
INFO 2025-10-18 15:37:49,983 basehttp 13280 17684 "GET /static/admin/js/vendor/select2/i18n/en.js HTTP/1.1" 200 844
INFO 2025-10-18 15:37:49,998 basehttp 13280 1228 "GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 200 9777
INFO 2025-10-18 15:37:49,998 basehttp 13280 23816 "GET /static/admin/js/prepopulate.js HTTP/1.1" 200 1531
INFO 2025-10-18 15:37:50,010 basehttp 13280 2636 "GET /static/admin/css/widgets.css HTTP/1.1" 200 11973
INFO 2025-10-18 15:37:50,016 basehttp 13280 22228 "GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 200 325171
INFO 2025-10-18 15:37:50,016 basehttp 13280 17684 "GET /static/admin/js/urlify.js HTTP/1.1" 200 7887
INFO 2025-10-18 15:37:50,034 basehttp 13280 17684 "GET /static/admin/js/prepopulate_init.js HTTP/1.1" 200 586
INFO 2025-10-18 15:37:50,053 basehttp 13280 17684 "GET /static/admin/img/icon-viewlink.svg HTTP/1.1" 200 581
INFO 2025-10-18 15:37:50,059 basehttp 13280 17684 "GET /static/admin/img/icon-deletelink.svg HTTP/1.1" 200 392
INFO 2025-10-18 15:37:50,065 basehttp 13280 17684 "GET /static/admin/img/icon-unknown.svg HTTP/1.1" 200 655
INFO 2025-10-18 15:37:50,070 basehttp 13280 17684 "GET /static/admin/js/change_form.js HTTP/1.1" 200 606
INFO 2025-10-18 15:37:50,114 basehttp 13280 21952 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
INFO 2025-10-18 15:37:50,315 basehttp 13280 21952 "GET /static/admin/img/inline-delete.svg HTTP/1.1" 200 537
ERROR 2025-10-18 15:37:50,506 log 13280 21952 Internal Server Error: /favicon.ico
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\loader.py", line 47, in select_template
    raise TemplateDoesNotExist(", ".join(template_name_list), chain=chain)
django.template.exceptions.TemplateDoesNotExist: index.html
ERROR 2025-10-18 15:37:50,511 basehttp 13280 21952 "GET /favicon.ico HTTP/1.1" 500 92367
INFO 2025-10-18 15:39:55,413 autoreload 13280 9436 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\pristineprimer\settings.py changed, reloading.
INFO 2025-10-18 15:39:57,479 autoreload 15464 15688 Watching for file changes with StatReloader
INFO 2025-10-18 15:40:59,933 autoreload 15464 15688 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\properties\urls.py changed, reloading.
INFO 2025-10-18 15:41:01,471 autoreload 11544 21912 Watching for file changes with StatReloader
INFO 2025-10-18 15:41:11,868 autoreload 11544 21912 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\properties\urls.py changed, reloading.
INFO 2025-10-18 15:41:13,290 autoreload 588 7680 Watching for file changes with StatReloader
INFO 2025-10-18 15:42:02,736 autoreload 588 7680 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\pristineprimer\urls.py changed, reloading.
INFO 2025-10-18 15:42:04,470 autoreload 16596 1436 Watching for file changes with StatReloader
INFO 2025-10-18 15:42:17,641 autoreload 16596 1436 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\properties\urls.py changed, reloading.
INFO 2025-10-18 15:42:19,373 autoreload 7112 6408 Watching for file changes with StatReloader
INFO 2025-10-18 15:42:23,217 autoreload 7112 6408 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\pristineprimer\urls.py changed, reloading.
INFO 2025-10-18 15:42:24,755 autoreload 15192 10584 Watching for file changes with StatReloader
INFO 2025-10-18 15:48:38,685 basehttp 15192 21440 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 15:48:39,001 basehttp 15192 16700 "OPTIONS /api/properties/1/ HTTP/1.1" 200 0
INFO 2025-10-18 15:48:39,246 basehttp 15192 21440 "GET /api/properties/1/ HTTP/1.1" 200 2055
INFO 2025-10-18 15:48:48,709 basehttp 15192 11988 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 15:48:49,029 basehttp 15192 23328 "OPTIONS /api/properties/? HTTP/1.1" 200 0
INFO 2025-10-18 15:48:49,205 basehttp 15192 11988 "GET /api/properties/? HTTP/1.1" 200 1944
INFO 2025-10-18 15:49:04,945 basehttp 15192 3264 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 15:49:05,175 basehttp 15192 3264 "GET /api/properties/1/ HTTP/1.1" 200 2055
INFO 2025-10-18 16:52:22,218 basehttp 15192 10712 "OPTIONS /api/auth/me/ HTTP/1.1" 200 0
INFO 2025-10-18 16:52:22,251 basehttp 15192 25544 "GET /api/auth/me/ HTTP/1.1" 200 32
INFO 2025-10-18 16:52:22,604 basehttp 15192 14944 "GET /api/properties/1/ HTTP/1.1" 200 2055
INFO 2025-10-18 16:52:27,894 basehttp 15192 16236 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 16:52:28,203 basehttp 15192 21152 "OPTIONS /api/properties/?featured=true&limit=6 HTTP/1.1" 200 0
INFO 2025-10-18 16:52:28,364 basehttp 15192 16236 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 1056
INFO 2025-10-18 16:52:38,789 basehttp 15192 18148 "GET /admin/properties/property/add/ HTTP/1.1" 200 101741
ERROR 2025-10-18 16:52:39,522 log 15192 18148 Internal Server Error: /favicon.ico
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\loader.py", line 47, in select_template
    raise TemplateDoesNotExist(", ".join(template_name_list), chain=chain)
django.template.exceptions.TemplateDoesNotExist: index.html
ERROR 2025-10-18 16:52:39,532 basehttp 15192 18148 "GET /favicon.ico HTTP/1.1" 500 92503
INFO 2025-10-18 16:59:22,087 basehttp 15192 14988 "GET /admin/ HTTP/1.1" 200 20863
ERROR 2025-10-18 16:59:22,328 log 15192 14988 Internal Server Error: /favicon.ico
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\loader.py", line 47, in select_template
    raise TemplateDoesNotExist(", ".join(template_name_list), chain=chain)
django.template.exceptions.TemplateDoesNotExist: index.html
ERROR 2025-10-18 16:59:22,334 basehttp 15192 14988 "GET /favicon.ico HTTP/1.1" 500 92479
INFO 2025-10-18 16:59:33,645 basehttp 15192 17288 "GET /admin/properties/property/add/ HTTP/1.1" 200 101741
INFO 2025-10-18 16:59:33,958 basehttp 15192 17288 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
ERROR 2025-10-18 16:59:34,203 log 15192 17288 Internal Server Error: /favicon.ico
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\loader.py", line 47, in select_template
    raise TemplateDoesNotExist(", ".join(template_name_list), chain=chain)
django.template.exceptions.TemplateDoesNotExist: index.html
ERROR 2025-10-18 16:59:34,209 basehttp 15192 17288 "GET /favicon.ico HTTP/1.1" 500 92503
INFO 2025-10-18 17:05:22,625 basehttp 15192 24844 "GET /admin/autocomplete/?app_label=properties&model_name=propertyamenity&field_name=amenity HTTP/1.1" 200 46
INFO 2025-10-18 17:05:26,115 basehttp 15192 24844 "GET /admin/autocomplete/?app_label=properties&model_name=propertyamenity&field_name=amenity HTTP/1.1" 200 46
INFO 2025-10-18 17:05:28,245 basehttp 15192 24844 "GET /admin/autocomplete/?app_label=properties&model_name=propertyamenity&field_name=amenity HTTP/1.1" 200 46
INFO 2025-10-18 17:05:30,949 basehttp 15192 24844 "GET /admin/autocomplete/?term=ho&app_label=properties&model_name=propertyamenity&field_name=amenity HTTP/1.1" 200 46
INFO 2025-10-18 17:05:31,802 basehttp 15192 24844 "GET /admin/autocomplete/?term=hos&app_label=properties&model_name=propertyamenity&field_name=amenity HTTP/1.1" 200 46
INFO 2025-10-18 17:05:32,179 basehttp 15192 24844 - Broken pipe from ('127.0.0.1', 55465)
INFO 2025-10-18 17:05:32,480 basehttp 15192 23580 - Broken pipe from ('127.0.0.1', 55479)
INFO 2025-10-18 17:05:32,805 basehttp 15192 20680 - Broken pipe from ('127.0.0.1', 55481)
INFO 2025-10-18 17:05:33,053 basehttp 15192 3532 - Broken pipe from ('127.0.0.1', 55484)
INFO 2025-10-18 17:05:33,346 basehttp 15192 18512 "GET /admin/autocomplete/?term=hospital&app_label=properties&model_name=propertyamenity&field_name=amenity HTTP/1.1" 200 46
INFO 2025-10-18 17:05:37,163 basehttp 15192 18512 "GET /admin/properties/amenity/add/?_to_field=id&_popup=1 HTTP/1.1" 200 7383
INFO 2025-10-18 17:05:37,283 basehttp 15192 18512 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
ERROR 2025-10-18 17:05:37,465 log 15192 18512 Internal Server Error: /favicon.ico
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\loader.py", line 47, in select_template
    raise TemplateDoesNotExist(", ".join(template_name_list), chain=chain)
django.template.exceptions.TemplateDoesNotExist: index.html
ERROR 2025-10-18 17:05:37,469 basehttp 15192 18512 "GET /favicon.ico HTTP/1.1" 500 92528
INFO 2025-10-18 17:06:01,772 basehttp 15192 22444 "POST /admin/properties/amenity/add/?_to_field=id&_popup=1 HTTP/1.1" 200 349
INFO 2025-10-18 17:06:01,936 basehttp 15192 22444 "GET /static/admin/js/popup_response.js HTTP/1.1" 200 532
INFO 2025-10-18 17:06:03,618 basehttp 15192 22444 "GET /admin/autocomplete/?app_label=properties&model_name=propertyamenity&field_name=amenity HTTP/1.1" 200 98
INFO 2025-10-18 17:09:52,094 basehttp 15192 21400 "POST /admin/properties/property/add/ HTTP/1.1" 200 105275
WARNING 2025-10-18 17:09:52,195 log 15192 25064 Not Found: /media/images (5).jpeg
WARNING 2025-10-18 17:09:52,201 basehttp 15192 25064 "GET /media/images%20(5).jpeg HTTP/1.1" 404 3289
WARNING 2025-10-18 17:09:52,219 log 15192 21728 Not Found: /media/download.jpeg
WARNING 2025-10-18 17:09:52,234 basehttp 15192 21728 "GET /media/download.jpeg HTTP/1.1" 404 3279
INFO 2025-10-18 17:09:52,288 basehttp 15192 21400 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
ERROR 2025-10-18 17:09:52,688 log 15192 21400 Internal Server Error: /favicon.ico
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\loader.py", line 47, in select_template
    raise TemplateDoesNotExist(", ".join(template_name_list), chain=chain)
django.template.exceptions.TemplateDoesNotExist: index.html
ERROR 2025-10-18 17:09:52,700 basehttp 15192 21400 "GET /favicon.ico HTTP/1.1" 500 92503
INFO 2025-10-18 17:10:40,826 basehttp 15192 25148 "POST /admin/properties/property/add/ HTTP/1.1" 200 105675
INFO 2025-10-18 17:10:40,951 basehttp 15192 25148 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
ERROR 2025-10-18 17:10:41,207 log 15192 25148 Internal Server Error: /favicon.ico
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\loader.py", line 47, in select_template
    raise TemplateDoesNotExist(", ".join(template_name_list), chain=chain)
django.template.exceptions.TemplateDoesNotExist: index.html
ERROR 2025-10-18 17:10:41,211 basehttp 15192 25148 "GET /favicon.ico HTTP/1.1" 500 92503
INFO 2025-10-18 17:13:17,790 basehttp 15192 9192 "POST /admin/properties/property/add/ HTTP/1.1" 302 0
INFO 2025-10-18 17:13:17,931 basehttp 15192 9192 "GET /admin/properties/property/ HTTP/1.1" 200 35470
INFO 2025-10-18 17:13:17,974 basehttp 15192 20360 "GET /static/admin/img/icon-yes.svg HTTP/1.1" 200 436
INFO 2025-10-18 17:13:17,974 basehttp 15192 9192 "GET /static/admin/css/changelists.css HTTP/1.1" 200 6878
INFO 2025-10-18 17:13:17,987 basehttp 15192 14696 "GET /static/admin/img/search.svg HTTP/1.1" 200 458
INFO 2025-10-18 17:13:17,987 basehttp 15192 9192 "GET /static/admin/img/icon-no.svg HTTP/1.1" 200 560
INFO 2025-10-18 17:13:18,002 basehttp 15192 9192 "GET /static/admin/js/filters.js HTTP/1.1" 200 978
INFO 2025-10-18 17:13:18,088 basehttp 15192 6896 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
INFO 2025-10-18 17:13:18,128 basehttp 15192 9192 "GET /static/admin/img/tooltag-add.svg HTTP/1.1" 200 331
INFO 2025-10-18 17:13:18,128 basehttp 15192 6896 "GET /static/admin/img/sorting-icons.svg HTTP/1.1" 200 1097
ERROR 2025-10-18 17:13:18,391 log 15192 6896 Internal Server Error: /favicon.ico
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\loader.py", line 47, in select_template
    raise TemplateDoesNotExist(", ".join(template_name_list), chain=chain)
django.template.exceptions.TemplateDoesNotExist: index.html
ERROR 2025-10-18 17:13:18,499 basehttp 15192 6896 "GET /favicon.ico HTTP/1.1" 500 92499
INFO 2025-10-18 17:13:25,459 basehttp 15192 22812 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 17:13:25,462 basehttp 15192 22812 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 17:13:25,658 basehttp 15192 22812 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 1755
INFO 2025-10-18 17:13:25,662 basehttp 15192 22812 "GET /api/auth/me/ HTTP/1.1" 200 32
INFO 2025-10-18 17:13:25,721 basehttp 15192 22812 "GET /media/property_images/2025/10/18/images_5.jpeg HTTP/1.1" 200 11615
INFO 2025-10-18 17:13:47,057 basehttp 15192 4916 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 17:13:47,371 basehttp 15192 3040 "OPTIONS /api/properties/5/ HTTP/1.1" 200 0
INFO 2025-10-18 17:13:47,519 basehttp 15192 4916 "GET /api/properties/5/ HTTP/1.1" 200 4336
INFO 2025-10-18 17:13:47,538 basehttp 15192 20636 "GET /media/property_images/2025/10/18/images_5.jpeg HTTP/1.1" 304 0
INFO 2025-10-18 17:13:47,539 basehttp 15192 4916 "GET /media/property_images/2025/10/18/download_4.jpeg HTTP/1.1" 200 11102
INFO 2025-10-18 17:14:07,521 basehttp 15192 8828 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 17:14:07,690 basehttp 15192 8828 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 1755
INFO 2025-10-18 17:14:17,322 basehttp 15192 17348 "GET /admin/properties/property/5/change/ HTTP/1.1" 200 116397
INFO 2025-10-18 17:14:17,374 basehttp 15192 9072 "GET /media/property_images/2025/10/18/images_5.jpeg HTTP/1.1" 200 11615
INFO 2025-10-18 17:14:17,380 basehttp 15192 3696 "GET /media/property_images/2025/10/18/download_4.jpeg HTTP/1.1" 200 11102
INFO 2025-10-18 17:14:17,449 basehttp 15192 17348 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
ERROR 2025-10-18 17:14:17,781 log 15192 17348 Internal Server Error: /favicon.ico
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\loader.py", line 47, in select_template
    raise TemplateDoesNotExist(", ".join(template_name_list), chain=chain)
django.template.exceptions.TemplateDoesNotExist: index.html
ERROR 2025-10-18 17:14:17,785 basehttp 15192 17348 "GET /favicon.ico HTTP/1.1" 500 92508
INFO 2025-10-18 17:15:22,060 basehttp 15192 11980 "POST /admin/properties/property/5/change/ HTTP/1.1" 200 116668
INFO 2025-10-18 17:15:22,101 basehttp 15192 244 "GET /media/property_images/2025/10/18/images_5.jpeg HTTP/1.1" 304 0
INFO 2025-10-18 17:15:22,103 basehttp 15192 23604 "GET /media/property_images/2025/10/18/download_4.jpeg HTTP/1.1" 304 0
INFO 2025-10-18 17:15:22,303 basehttp 15192 11980 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
ERROR 2025-10-18 17:15:22,508 log 15192 11980 Internal Server Error: /favicon.ico
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\loader.py", line 47, in select_template
    raise TemplateDoesNotExist(", ".join(template_name_list), chain=chain)
django.template.exceptions.TemplateDoesNotExist: index.html
ERROR 2025-10-18 17:15:22,512 basehttp 15192 11980 "GET /favicon.ico HTTP/1.1" 500 92508
INFO 2025-10-18 17:15:57,509 basehttp 15192 24304 "POST /admin/properties/property/5/change/ HTTP/1.1" 200 116858
INFO 2025-10-18 17:15:57,565 basehttp 15192 1816 "GET /media/property_images/2025/10/18/images_5.jpeg HTTP/1.1" 304 0
INFO 2025-10-18 17:15:57,566 basehttp 15192 23944 "GET /media/property_images/2025/10/18/download_4.jpeg HTTP/1.1" 304 0
INFO 2025-10-18 17:15:57,650 basehttp 15192 24304 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
ERROR 2025-10-18 17:15:57,868 log 15192 24304 Internal Server Error: /favicon.ico
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\loader.py", line 47, in select_template
    raise TemplateDoesNotExist(", ".join(template_name_list), chain=chain)
django.template.exceptions.TemplateDoesNotExist: index.html
ERROR 2025-10-18 17:15:57,872 basehttp 15192 24304 "GET /favicon.ico HTTP/1.1" 500 92508
INFO 2025-10-18 17:17:15,186 basehttp 15192 18128 "POST /admin/properties/property/5/change/ HTTP/1.1" 200 116732
INFO 2025-10-18 17:17:15,229 basehttp 15192 21416 "GET /media/property_images/2025/10/18/images_5.jpeg HTTP/1.1" 304 0
INFO 2025-10-18 17:17:15,231 basehttp 15192 19856 "GET /media/property_images/2025/10/18/download_4.jpeg HTTP/1.1" 304 0
INFO 2025-10-18 17:17:15,300 basehttp 15192 18128 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
ERROR 2025-10-18 17:17:15,506 log 15192 18128 Internal Server Error: /favicon.ico
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\loader.py", line 47, in select_template
    raise TemplateDoesNotExist(", ".join(template_name_list), chain=chain)
django.template.exceptions.TemplateDoesNotExist: index.html
ERROR 2025-10-18 17:17:15,511 basehttp 15192 18128 "GET /favicon.ico HTTP/1.1" 500 92508
INFO 2025-10-18 17:17:42,898 basehttp 15192 19636 "POST /admin/properties/property/5/change/ HTTP/1.1" 302 0
INFO 2025-10-18 17:17:43,015 basehttp 15192 19636 "GET /admin/properties/property/ HTTP/1.1" 200 35475
INFO 2025-10-18 17:17:43,279 basehttp 15192 19636 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
ERROR 2025-10-18 17:17:43,485 log 15192 19636 Internal Server Error: /favicon.ico
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\loader.py", line 47, in select_template
    raise TemplateDoesNotExist(", ".join(template_name_list), chain=chain)
django.template.exceptions.TemplateDoesNotExist: index.html
ERROR 2025-10-18 17:17:43,491 basehttp 15192 19636 "GET /favicon.ico HTTP/1.1" 500 92499
INFO 2025-10-18 17:17:52,513 basehttp 15192 1652 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 17:17:52,518 basehttp 15192 1652 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 17:17:52,830 basehttp 15192 14676 "GET /api/auth/me/ HTTP/1.1" 200 32
INFO 2025-10-18 17:17:52,832 basehttp 15192 1652 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 1755
INFO 2025-10-18 17:17:52,915 basehttp 15192 1652 "GET /media/property_images/2025/10/18/images_5.jpeg HTTP/1.1" 304 0
INFO 2025-10-18 17:18:00,414 basehttp 15192 1652 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 17:18:00,414 basehttp 15192 14676 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 17:18:00,800 basehttp 15192 3276 "OPTIONS /api/auth/me/ HTTP/1.1" 200 0
INFO 2025-10-18 17:18:00,801 basehttp 15192 3712 "OPTIONS /api/properties/?featured=true&limit=6 HTTP/1.1" 200 0
INFO 2025-10-18 17:18:00,839 basehttp 15192 14676 "GET /api/auth/me/ HTTP/1.1" 200 32
INFO 2025-10-18 17:18:00,992 basehttp 15192 1652 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 1755
INFO 2025-10-18 17:18:21,124 basehttp 15192 11680 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 17:18:21,270 basehttp 15192 11680 "GET /api/properties/5/ HTTP/1.1" 200 4353
INFO 2025-10-18 17:18:21,289 basehttp 15192 11680 "GET /media/property_images/2025/10/18/download_4.jpeg HTTP/1.1" 304 0
INFO 2025-10-18 17:18:36,903 basehttp 15192 9088 "GET /admin/properties/property/5/change/ HTTP/1.1" 200 116494
INFO 2025-10-18 17:18:36,946 basehttp 15192 11120 "GET /media/property_images/2025/10/18/images_5.jpeg HTTP/1.1" 304 0
INFO 2025-10-18 17:18:36,947 basehttp 15192 24704 "GET /media/property_images/2025/10/18/download_4.jpeg HTTP/1.1" 304 0
INFO 2025-10-18 17:18:37,020 basehttp 15192 9088 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
ERROR 2025-10-18 17:18:37,253 log 15192 9088 Internal Server Error: /favicon.ico
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\loader.py", line 47, in select_template
    raise TemplateDoesNotExist(", ".join(template_name_list), chain=chain)
django.template.exceptions.TemplateDoesNotExist: index.html
ERROR 2025-10-18 17:18:37,260 basehttp 15192 9088 "GET /favicon.ico HTTP/1.1" 500 92508
INFO 2025-10-18 17:18:41,151 basehttp 15192 9088 "GET /admin/properties/property/ HTTP/1.1" 200 35246
ERROR 2025-10-18 17:18:41,346 log 15192 9088 Internal Server Error: /favicon.ico
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\loader.py", line 47, in select_template
    raise TemplateDoesNotExist(", ".join(template_name_list), chain=chain)
django.template.exceptions.TemplateDoesNotExist: index.html
ERROR 2025-10-18 17:18:41,350 basehttp 15192 9088 "GET /favicon.ico HTTP/1.1" 500 92499
INFO 2025-10-18 17:18:43,900 basehttp 15192 22796 "GET /admin/properties/property/5/change/ HTTP/1.1" 200 116494
INFO 2025-10-18 17:18:44,014 basehttp 15192 22796 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
ERROR 2025-10-18 17:18:44,239 log 15192 22796 Internal Server Error: /favicon.ico
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\loader.py", line 47, in select_template
    raise TemplateDoesNotExist(", ".join(template_name_list), chain=chain)
django.template.exceptions.TemplateDoesNotExist: index.html
ERROR 2025-10-18 17:18:44,245 basehttp 15192 22796 "GET /favicon.ico HTTP/1.1" 500 92508
INFO 2025-10-18 17:19:03,326 basehttp 15192 16108 "GET /admin/properties/property/ HTTP/1.1" 200 35246
ERROR 2025-10-18 17:19:03,531 log 15192 16108 Internal Server Error: /favicon.ico
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\loader.py", line 47, in select_template
    raise TemplateDoesNotExist(", ".join(template_name_list), chain=chain)
django.template.exceptions.TemplateDoesNotExist: index.html
ERROR 2025-10-18 17:19:03,534 basehttp 15192 16108 "GET /favicon.ico HTTP/1.1" 500 92499
INFO 2025-10-18 17:19:17,190 basehttp 15192 19924 "GET /admin/properties/property/5/change/ HTTP/1.1" 200 116494
INFO 2025-10-18 17:19:17,240 basehttp 15192 24932 "GET /media/property_images/2025/10/18/images_5.jpeg HTTP/1.1" 304 0
INFO 2025-10-18 17:19:17,245 basehttp 15192 1940 "GET /media/property_images/2025/10/18/download_4.jpeg HTTP/1.1" 304 0
INFO 2025-10-18 17:19:17,317 basehttp 15192 19924 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
ERROR 2025-10-18 17:19:17,604 log 15192 19924 Internal Server Error: /favicon.ico
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\loader.py", line 47, in select_template
    raise TemplateDoesNotExist(", ".join(template_name_list), chain=chain)
django.template.exceptions.TemplateDoesNotExist: index.html
ERROR 2025-10-18 17:19:17,610 basehttp 15192 19924 "GET /favicon.ico HTTP/1.1" 500 92508
INFO 2025-10-18 17:20:49,843 basehttp 15192 19608 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 17:20:49,993 basehttp 15192 19608 "GET /api/properties/? HTTP/1.1" 200 2643
INFO 2025-10-18 17:20:58,882 basehttp 15192 19608 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 17:20:59,113 basehttp 15192 19608 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 1755
INFO 2025-10-18 17:21:33,981 basehttp 15192 22876 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 17:21:34,236 basehttp 15192 22876 "GET /api/properties/5/ HTTP/1.1" 200 4353
INFO 2025-10-18 17:21:34,261 basehttp 15192 22876 "GET /media/property_images/2025/10/18/download_4.jpeg HTTP/1.1" 304 0
INFO 2025-10-18 17:23:34,376 basehttp 15192 18052 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 17:23:34,762 basehttp 15192 18052 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 1755
INFO 2025-10-18 17:23:44,270 basehttp 15192 8560 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 17:23:44,470 basehttp 15192 8560 "GET /api/properties/5/ HTTP/1.1" 200 4353
INFO 2025-10-18 17:23:44,492 basehttp 15192 8560 "GET /media/property_images/2025/10/18/download_4.jpeg HTTP/1.1" 304 0
INFO 2025-10-18 17:23:48,596 basehttp 15192 20612 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 17:23:48,798 basehttp 15192 20612 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 1755
INFO 2025-10-18 17:23:48,870 basehttp 15192 20612 "GET /media/property_images/2025/10/18/images_5.jpeg HTTP/1.1" 304 0
INFO 2025-10-18 17:33:49,258 basehttp 15192 15120 "GET /admin/properties/property/5/change/ HTTP/1.1" 200 116494
ERROR 2025-10-18 17:33:49,877 log 15192 15120 Internal Server Error: /favicon.ico
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\loader.py", line 47, in select_template
    raise TemplateDoesNotExist(", ".join(template_name_list), chain=chain)
django.template.exceptions.TemplateDoesNotExist: index.html
ERROR 2025-10-18 17:33:49,880 basehttp 15192 15120 "GET /favicon.ico HTTP/1.1" 500 92508
ERROR 2025-10-18 17:33:52,910 log 15192 15120 Internal Server Error: /
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\loader.py", line 47, in select_template
    raise TemplateDoesNotExist(", ".join(template_name_list), chain=chain)
django.template.exceptions.TemplateDoesNotExist: index.html
ERROR 2025-10-18 17:33:52,917 basehttp 15192 15120 "GET / HTTP/1.1" 500 92570
ERROR 2025-10-18 17:33:53,226 log 15192 15120 Internal Server Error: /favicon.ico
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\loader.py", line 47, in select_template
    raise TemplateDoesNotExist(", ".join(template_name_list), chain=chain)
django.template.exceptions.TemplateDoesNotExist: index.html
ERROR 2025-10-18 17:33:53,232 basehttp 15192 15120 "GET /favicon.ico HTTP/1.1" 500 92473
INFO 2025-10-18 17:35:39,085 basehttp 15192 21808 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 17:35:39,102 basehttp 15192 21808 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 17:35:39,332 basehttp 15192 19284 "GET /api/auth/me/ HTTP/1.1" 200 32
INFO 2025-10-18 17:35:39,434 basehttp 15192 21808 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 1755
INFO 2025-10-18 17:35:39,540 basehttp 15192 21808 "GET /media/property_images/2025/10/18/images_5.jpeg HTTP/1.1" 304 0
INFO 2025-10-18 17:35:44,460 basehttp 15192 21808 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 17:35:44,482 basehttp 15192 19284 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 17:35:44,915 basehttp 15192 21336 "OPTIONS /api/properties/?featured=true&limit=6 HTTP/1.1" 200 0
INFO 2025-10-18 17:35:44,915 basehttp 15192 11348 "OPTIONS /api/auth/me/ HTTP/1.1" 200 0
INFO 2025-10-18 17:35:44,934 basehttp 15192 21808 "GET /api/auth/me/ HTTP/1.1" 200 32
INFO 2025-10-18 17:35:45,079 basehttp 15192 19284 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 1755
INFO 2025-10-18 17:41:16,499 autoreload 15192 10584 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\properties\serializers.py changed, reloading.
INFO 2025-10-18 17:41:19,297 autoreload 19396 344 Watching for file changes with StatReloader
INFO 2025-10-18 17:41:34,538 autoreload 19396 344 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\properties\serializers.py changed, reloading.
INFO 2025-10-18 17:41:35,969 autoreload 22636 13172 Watching for file changes with StatReloader
INFO 2025-10-18 17:44:16,770 basehttp 22636 21068 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 17:44:16,777 basehttp 22636 21068 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 17:44:17,005 basehttp 22636 24088 "GET /api/auth/me/ HTTP/1.1" 200 32
INFO 2025-10-18 17:44:17,106 basehttp 22636 21068 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 1755
INFO 2025-10-18 17:44:17,181 basehttp 22636 21068 "GET /media/property_images/2025/10/18/images_5.jpeg HTTP/1.1" 304 0
INFO 2025-10-18 17:44:51,144 basehttp 22636 740 "OPTIONS /api/properties/2/favorite/ HTTP/1.1" 200 0
WARNING 2025-10-18 17:44:51,461 log 22636 2288 Forbidden: /api/properties/2/favorite/
WARNING 2025-10-18 17:44:51,462 basehttp 22636 2288 "POST /api/properties/2/favorite/ HTTP/1.1" 403 58
INFO 2025-10-18 17:47:26,455 autoreload 22636 13172 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\properties\serializers.py changed, reloading.
INFO 2025-10-18 17:47:27,907 autoreload 15620 2404 Watching for file changes with StatReloader
INFO 2025-10-18 17:47:38,532 basehttp 15620 1236 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 17:47:38,547 basehttp 15620 16712 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 17:47:38,593 basehttp 15620 16712 "GET /api/auth/me/ HTTP/1.1" 200 32
ERROR 2025-10-18 17:47:38,848 log 15620 1236 Internal Server Error: /api/properties/
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 515, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 475, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 486, in raise_uncaught_exception
    raise exc
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 512, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\mixins.py", line 43, in list
    return self.get_paginated_response(serializer.data)
                                       ^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 797, in data
    ret = super().data
          ^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 251, in data
    self._data = self.to_representation(self.instance)
                 ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 716, in to_representation
    self.child.to_representation(item) for item in iterable
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 525, in to_representation
    for field in fields:
                 ^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 386, in _readable_fields
    for field in self.fields.values():
                 ^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\utils\functional.py", line 47, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
                                         ~~~~~~~~~^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 374, in fields
    for key, value in self.get_fields().items():
                      ~~~~~~~~~~~~~~~^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 1105, in get_fields
    field_class, field_kwargs = self.build_field(
                                ~~~~~~~~~~~~~~~~^
        source, info, model, depth
        ^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 1251, in build_field
    return self.build_unknown_field(field_name, model_class)
           ~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 1369, in build_unknown_field
    raise ImproperlyConfigured(
    ...<2 lines>...
    )
django.core.exceptions.ImproperlyConfigured: Field name `is_favorited` is not valid for model `Property` in `properties.serializers.PropertyListSerializer`.
ERROR 2025-10-18 17:47:38,864 basehttp 15620 1236 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 500 168860
INFO 2025-10-18 17:48:03,064 basehttp 15620 1236 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 17:48:03,066 basehttp 15620 16712 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 17:48:03,281 basehttp 15620 1236 "GET /api/auth/me/ HTTP/1.1" 200 32
ERROR 2025-10-18 17:48:03,647 log 15620 16712 Internal Server Error: /api/properties/
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 515, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 475, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 486, in raise_uncaught_exception
    raise exc
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 512, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\mixins.py", line 43, in list
    return self.get_paginated_response(serializer.data)
                                       ^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 797, in data
    ret = super().data
          ^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 251, in data
    self._data = self.to_representation(self.instance)
                 ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 716, in to_representation
    self.child.to_representation(item) for item in iterable
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 525, in to_representation
    for field in fields:
                 ^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 386, in _readable_fields
    for field in self.fields.values():
                 ^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\utils\functional.py", line 47, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
                                         ~~~~~~~~~^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 374, in fields
    for key, value in self.get_fields().items():
                      ~~~~~~~~~~~~~~~^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 1105, in get_fields
    field_class, field_kwargs = self.build_field(
                                ~~~~~~~~~~~~~~~~^
        source, info, model, depth
        ^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 1251, in build_field
    return self.build_unknown_field(field_name, model_class)
           ~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 1369, in build_unknown_field
    raise ImproperlyConfigured(
    ...<2 lines>...
    )
django.core.exceptions.ImproperlyConfigured: Field name `is_favorited` is not valid for model `Property` in `properties.serializers.PropertyListSerializer`.
ERROR 2025-10-18 17:48:03,663 basehttp 15620 16712 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 500 168860
INFO 2025-10-18 17:51:29,636 autoreload 15620 2404 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\properties\serializers.py changed, reloading.
INFO 2025-10-18 17:51:31,107 autoreload 7432 17060 Watching for file changes with StatReloader
INFO 2025-10-18 17:51:44,385 basehttp 7432 22128 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 17:51:44,394 basehttp 7432 22128 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 17:51:44,614 basehttp 7432 16512 "GET /api/auth/me/ HTTP/1.1" 200 32
ERROR 2025-10-18 17:51:44,723 log 7432 22128 Internal Server Error: /api/properties/
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 515, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 475, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 486, in raise_uncaught_exception
    raise exc
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 512, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\mixins.py", line 43, in list
    return self.get_paginated_response(serializer.data)
                                       ^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 797, in data
    ret = super().data
          ^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 251, in data
    self._data = self.to_representation(self.instance)
                 ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 716, in to_representation
    self.child.to_representation(item) for item in iterable
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 525, in to_representation
    for field in fields:
                 ^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 386, in _readable_fields
    for field in self.fields.values():
                 ^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\utils\functional.py", line 47, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
                                         ~~~~~~~~~^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 374, in fields
    for key, value in self.get_fields().items():
                      ~~~~~~~~~~~~~~~^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 1105, in get_fields
    field_class, field_kwargs = self.build_field(
                                ~~~~~~~~~~~~~~~~^
        source, info, model, depth
        ^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 1251, in build_field
    return self.build_unknown_field(field_name, model_class)
           ~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 1369, in build_unknown_field
    raise ImproperlyConfigured(
    ...<2 lines>...
    )
django.core.exceptions.ImproperlyConfigured: Field name `is_favorited` is not valid for model `Property` in `properties.serializers.PropertyListSerializer`.
ERROR 2025-10-18 17:51:44,738 basehttp 7432 22128 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 500 168860
INFO 2025-10-18 17:51:48,424 basehttp 7432 22128 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 17:51:48,426 basehttp 7432 15508 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 17:51:48,591 basehttp 7432 22128 "GET /api/auth/me/ HTTP/1.1" 200 32
ERROR 2025-10-18 17:51:48,920 log 7432 15508 Internal Server Error: /api/properties/
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 515, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 475, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 486, in raise_uncaught_exception
    raise exc
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 512, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\mixins.py", line 43, in list
    return self.get_paginated_response(serializer.data)
                                       ^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 797, in data
    ret = super().data
          ^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 251, in data
    self._data = self.to_representation(self.instance)
                 ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 716, in to_representation
    self.child.to_representation(item) for item in iterable
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 525, in to_representation
    for field in fields:
                 ^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 386, in _readable_fields
    for field in self.fields.values():
                 ^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\utils\functional.py", line 47, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
                                         ~~~~~~~~~^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 374, in fields
    for key, value in self.get_fields().items():
                      ~~~~~~~~~~~~~~~^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 1105, in get_fields
    field_class, field_kwargs = self.build_field(
                                ~~~~~~~~~~~~~~~~^
        source, info, model, depth
        ^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 1251, in build_field
    return self.build_unknown_field(field_name, model_class)
           ~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 1369, in build_unknown_field
    raise ImproperlyConfigured(
    ...<2 lines>...
    )
django.core.exceptions.ImproperlyConfigured: Field name `is_favorited` is not valid for model `Property` in `properties.serializers.PropertyListSerializer`.
ERROR 2025-10-18 17:51:48,956 basehttp 7432 15508 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 500 168860
INFO 2025-10-18 17:52:18,194 autoreload 7432 17060 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\properties\serializers.py changed, reloading.
INFO 2025-10-18 17:52:19,771 autoreload 14208 24564 Watching for file changes with StatReloader
INFO 2025-10-18 17:52:26,404 basehttp 14208 13028 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 17:52:26,404 basehttp 14208 23480 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 17:52:26,442 basehttp 14208 13028 "GET /api/auth/me/ HTTP/1.1" 200 32
INFO 2025-10-18 17:52:26,619 basehttp 14208 23480 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-18 17:52:26,675 basehttp 14208 23480 "GET /media/property_images/2025/10/18/images_5.jpeg HTTP/1.1" 304 0
INFO 2025-10-18 17:52:58,828 basehttp 14208 2908 "OPTIONS /api/properties/5/favorite/ HTTP/1.1" 200 0
WARNING 2025-10-18 17:52:59,152 log 14208 10900 Forbidden: /api/properties/5/favorite/
WARNING 2025-10-18 17:52:59,153 basehttp 14208 10900 "POST /api/properties/5/favorite/ HTTP/1.1" 403 58
INFO 2025-10-18 17:54:27,614 autoreload 14208 24564 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\properties\serializers.py changed, reloading.
INFO 2025-10-18 17:54:29,463 autoreload 22544 22676 Watching for file changes with StatReloader
INFO 2025-10-18 17:54:36,567 autoreload 22544 22676 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\properties\serializers.py changed, reloading.
INFO 2025-10-18 17:54:38,075 autoreload 20436 19664 Watching for file changes with StatReloader
INFO 2025-10-18 17:54:44,032 autoreload 20436 19664 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\properties\serializers.py changed, reloading.
INFO 2025-10-18 17:54:45,149 autoreload 25156 2136 Watching for file changes with StatReloader
INFO 2025-10-18 17:56:06,193 basehttp 25156 22300 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 17:56:06,358 basehttp 25156 22300 "GET /api/properties/5/ HTTP/1.1" 200 4353
INFO 2025-10-18 17:56:06,384 basehttp 25156 22300 "GET /media/property_images/2025/10/18/download_4.jpeg HTTP/1.1" 304 0
INFO 2025-10-18 17:56:23,393 basehttp 25156 4952 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 17:56:23,553 basehttp 25156 4952 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4188
INFO 2025-10-18 18:13:45,786 basehttp 25156 12264 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 18:13:45,798 basehttp 25156 12264 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 18:13:46,233 basehttp 25156 21704 "GET /api/auth/me/ HTTP/1.1" 200 32
INFO 2025-10-18 18:13:46,669 basehttp 25156 12264 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4188
INFO 2025-10-18 18:13:47,101 basehttp 25156 12264 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 18:13:47,388 basehttp 25156 12264 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4188
INFO 2025-10-18 18:15:09,001 basehttp 25156 9504 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 18:15:09,012 basehttp 25156 9504 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 18:15:09,377 basehttp 25156 2076 "GET /api/auth/me/ HTTP/1.1" 200 32
INFO 2025-10-18 18:15:09,561 basehttp 25156 9504 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4188
INFO 2025-10-18 18:15:16,946 basehttp 25156 20516 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 18:15:17,004 basehttp 25156 13776 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 18:15:17,038 basehttp 25156 13776 "GET /api/auth/me/ HTTP/1.1" 200 32
INFO 2025-10-18 18:15:17,221 basehttp 25156 20516 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4188
INFO 2025-10-18 18:15:17,302 basehttp 25156 20516 "GET /media/property_images/2025/10/18/images_5.jpeg HTTP/1.1" 304 0
INFO 2025-10-18 18:18:20,201 basehttp 25156 2780 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 18:18:20,508 basehttp 25156 2780 "GET /api/properties/5/ HTTP/1.1" 200 4353
INFO 2025-10-18 18:18:20,557 basehttp 25156 2780 "GET /media/property_images/2025/10/18/download_4.jpeg HTTP/1.1" 304 0
INFO 2025-10-18 18:18:28,526 basehttp 25156 8952 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 18:18:28,843 basehttp 25156 23172 "OPTIONS /api/properties/? HTTP/1.1" 200 0
INFO 2025-10-18 18:18:29,100 basehttp 25156 8952 "GET /api/properties/? HTTP/1.1" 200 6206
INFO 2025-10-18 18:18:35,554 basehttp 25156 16168 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 18:18:35,741 basehttp 25156 16168 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4188
INFO 2025-10-18 18:19:20,626 basehttp 25156 8560 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 18:19:20,855 basehttp 25156 8560 "GET /api/properties/5/ HTTP/1.1" 200 4353
INFO 2025-10-18 18:19:23,958 basehttp 25156 22564 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 18:19:24,311 basehttp 25156 22564 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4188
INFO 2025-10-18 18:20:17,651 basehttp 25156 14196 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 18:20:17,838 basehttp 25156 14196 "GET /api/properties/5/ HTTP/1.1" 200 4353
INFO 2025-10-18 18:20:19,396 basehttp 25156 14196 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 18:20:19,666 basehttp 25156 14196 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4188
INFO 2025-10-18 18:23:16,844 basehttp 25156 23336 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 18:23:17,181 basehttp 25156 22016 "OPTIONS /api/auth/login/ HTTP/1.1" 200 0
WARNING 2025-10-18 18:23:18,779 log 25156 19388 Unauthorized: /api/auth/login/
WARNING 2025-10-18 18:23:18,780 basehttp 25156 19388 "POST /api/auth/login/ HTTP/1.1" 401 52
INFO 2025-10-18 18:23:32,408 basehttp 25156 11696 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 18:23:32,416 basehttp 25156 11696 "GET /api/auth/me/ HTTP/1.1" 200 32
INFO 2025-10-18 18:23:41,064 basehttp 25156 3956 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 18:23:42,415 basehttp 25156 3956 "POST /api/auth/login/ HTTP/1.1" 200 918
INFO 2025-10-18 18:23:42,490 basehttp 25156 3956 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 18:23:42,500 basehttp 25156 3956 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 18:23:42,841 basehttp 25156 23060 "OPTIONS /api/auth/dashboard/overview/ HTTP/1.1" 200 0
INFO 2025-10-18 18:23:42,848 basehttp 25156 23480 "OPTIONS /api/properties/my_properties/ HTTP/1.1" 200 0
INFO 2025-10-18 18:23:43,071 basehttp 25156 10244 "GET /api/properties/my_properties/ HTTP/1.1" 200 4021
ERROR 2025-10-18 18:23:43,161 log 25156 3956 Internal Server Error: /api/auth/dashboard/overview/
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\views\generic\base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 515, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 475, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 486, in raise_uncaught_exception
    raise exc
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 512, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\decorators.py", line 50, in handler
    return func(*args, **kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\users\views.py", line 183, in dashboard_overview
    my_listings = Property.objects.filter(owner=user)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\query.py", line 1493, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\query.py", line 1511, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\query.py", line 1518, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
    ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\sql\query.py", line 1646, in add_q
    clause, _ = self._add_q(q_object, can_reuse)
                ~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\sql\query.py", line 1678, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ~~~~~~~~~~~~~~~~~^
        child,
        ^^^^^^
    ...<7 lines>...
        update_join_types=update_join_types,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\sql\query.py", line 1526, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg, summarize)
                                        ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\sql\query.py", line 1333, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
                                ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\sql\query.py", line 1806, in names_to_path
    raise FieldError(
    ...<2 lines>...
    )
django.core.exceptions.FieldError: Cannot resolve keyword 'owner' into field. Choices are: address, agent, agent_id, amenities, bathrooms, bedrooms, city, contact_info, created_at, deposit_percentage, description, discount_offers, distance_to_main_road, documents, electricity_availability, favorited_by, featured, has_beacons, has_borehole, has_central_air, has_drainage, has_fireplace, has_garage, has_garden, has_piped_water, has_pool, has_sewer_system, has_subdivision_approval, id, images, inquiries, inquiry_count, internet_availability, is_fenced, is_gated_community, is_negotiable, land_type, landmarks, latitude, longitude, lot_size, media, num_plots_available, payment_plan_details, plot_dimensions, price, price_per_unit, price_unit, property_type, published_at, road_access_type, seller, seller_id, short_description, size_acres, soil_type, square_feet, state, status, title, title_deed_status, topography, total_plots, updated_at, useractivity, views_count, water_supply_types, year_built, zip_code, zoning
ERROR 2025-10-18 18:23:43,187 basehttp 25156 3956 "GET /api/auth/dashboard/overview/ HTTP/1.1" 500 164392
INFO 2025-10-18 18:23:50,784 basehttp 25156 22196 "GET /api/auth/csrf/ HTTP/1.1" 200 81
ERROR 2025-10-18 18:23:51,043 log 25156 22196 Internal Server Error: /api/properties/
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 515, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 475, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 486, in raise_uncaught_exception
    raise exc
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 512, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\mixins.py", line 43, in list
    return self.get_paginated_response(serializer.data)
                                       ^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 797, in data
    ret = super().data
          ^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 251, in data
    self._data = self.to_representation(self.instance)
                 ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 716, in to_representation
    self.child.to_representation(item) for item in iterable
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 540, in to_representation
    ret[field.field_name] = field.to_representation(attribute)
                            ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\fields.py", line 1870, in to_representation
    return method(value)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\properties\serializers.py", line 123, in get_is_favorited
    return obj.favorites.filter(user=request.user).exists()
           ^^^^^^^^^^^^^
AttributeError: 'Property' object has no attribute 'favorites'. Did you mean: 'favorited_by'?
ERROR 2025-10-18 18:23:51,062 basehttp 25156 22196 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 500 172073
INFO 2025-10-18 18:30:33,412 basehttp 25156 9088 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 18:30:33,426 basehttp 25156 9088 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 18:30:33,849 basehttp 25156 12576 "GET /api/properties/my_properties/ HTTP/1.1" 200 4021
ERROR 2025-10-18 18:30:33,905 log 25156 9088 Internal Server Error: /api/auth/dashboard/overview/
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\views\generic\base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 515, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 475, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 486, in raise_uncaught_exception
    raise exc
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 512, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\decorators.py", line 50, in handler
    return func(*args, **kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\users\views.py", line 183, in dashboard_overview
    my_listings = Property.objects.filter(owner=user)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\query.py", line 1493, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\query.py", line 1511, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\query.py", line 1518, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
    ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\sql\query.py", line 1646, in add_q
    clause, _ = self._add_q(q_object, can_reuse)
                ~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\sql\query.py", line 1678, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ~~~~~~~~~~~~~~~~~^
        child,
        ^^^^^^
    ...<7 lines>...
        update_join_types=update_join_types,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\sql\query.py", line 1526, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg, summarize)
                                        ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\sql\query.py", line 1333, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
                                ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\sql\query.py", line 1806, in names_to_path
    raise FieldError(
    ...<2 lines>...
    )
django.core.exceptions.FieldError: Cannot resolve keyword 'owner' into field. Choices are: address, agent, agent_id, amenities, bathrooms, bedrooms, city, contact_info, created_at, deposit_percentage, description, discount_offers, distance_to_main_road, documents, electricity_availability, favorited_by, featured, has_beacons, has_borehole, has_central_air, has_drainage, has_fireplace, has_garage, has_garden, has_piped_water, has_pool, has_sewer_system, has_subdivision_approval, id, images, inquiries, inquiry_count, internet_availability, is_fenced, is_gated_community, is_negotiable, land_type, landmarks, latitude, longitude, lot_size, media, num_plots_available, payment_plan_details, plot_dimensions, price, price_per_unit, price_unit, property_type, published_at, road_access_type, seller, seller_id, short_description, size_acres, soil_type, square_feet, state, status, title, title_deed_status, topography, total_plots, updated_at, useractivity, views_count, water_supply_types, year_built, zip_code, zoning
ERROR 2025-10-18 18:30:33,936 basehttp 25156 9088 "GET /api/auth/dashboard/overview/ HTTP/1.1" 500 164392
INFO 2025-10-18 18:36:09,824 basehttp 25156 13596 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 18:36:10,151 basehttp 25156 2252 "OPTIONS /api/properties/ HTTP/1.1" 200 0
WARNING 2025-10-18 18:36:10,326 log 25156 13596 Forbidden: /api/properties/
WARNING 2025-10-18 18:36:10,327 basehttp 25156 13596 "POST /api/properties/ HTTP/1.1" 403 108
INFO 2025-10-18 18:36:10,337 basehttp 25156 13596 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 18:36:10,347 basehttp 25156 2252 "OPTIONS /api/properties/create/ HTTP/1.1" 200 0
WARNING 2025-10-18 18:36:10,471 log 25156 13596 Forbidden: /api/properties/create/
WARNING 2025-10-18 18:36:10,472 basehttp 25156 13596 "POST /api/properties/create/ HTTP/1.1" 403 108
INFO 2025-10-18 18:36:10,483 basehttp 25156 13596 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 18:36:10,491 basehttp 25156 2252 "OPTIONS /api/create/ HTTP/1.1" 200 0
WARNING 2025-10-18 18:36:10,603 log 25156 13596 Forbidden: /api/create/
WARNING 2025-10-18 18:36:10,604 basehttp 25156 13596 "POST /api/create/ HTTP/1.1" 403 108
INFO 2025-10-18 18:52:59,414 basehttp 25156 10632 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 18:52:59,961 basehttp 25156 10632 "GET /api/auth/me/ HTTP/1.1" 200 887
INFO 2025-10-18 18:54:49,756 basehttp 25156 8740 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 18:54:50,129 basehttp 25156 8740 "GET /api/auth/me/ HTTP/1.1" 200 887
INFO 2025-10-18 18:59:18,973 basehttp 25156 19792 "GET /api/auth/csrf/ HTTP/1.1" 200 81
WARNING 2025-10-18 18:59:19,142 log 25156 19792 Forbidden: /api/properties/
WARNING 2025-10-18 18:59:19,144 basehttp 25156 19792 "POST /api/properties/ HTTP/1.1" 403 108
INFO 2025-10-18 18:59:19,157 basehttp 25156 19792 "GET /api/auth/csrf/ HTTP/1.1" 200 81
WARNING 2025-10-18 18:59:19,283 log 25156 19792 Forbidden: /api/properties/create/
WARNING 2025-10-18 18:59:19,284 basehttp 25156 19792 "POST /api/properties/create/ HTTP/1.1" 403 108
INFO 2025-10-18 18:59:19,293 basehttp 25156 19792 "GET /api/auth/csrf/ HTTP/1.1" 200 81
WARNING 2025-10-18 18:59:19,389 log 25156 19792 Forbidden: /api/create/
WARNING 2025-10-18 18:59:19,390 basehttp 25156 19792 "POST /api/create/ HTTP/1.1" 403 108
INFO 2025-10-18 19:06:22,086 basehttp 25156 23152 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 19:06:22,359 basehttp 25156 23152 "GET /api/auth/me/ HTTP/1.1" 200 887
INFO 2025-10-18 19:06:29,641 basehttp 25156 23152 "GET /api/auth/csrf/ HTTP/1.1" 200 81
ERROR 2025-10-18 19:06:30,349 log 25156 23152 Internal Server Error: /api/properties/
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 515, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 475, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 486, in raise_uncaught_exception
    raise exc
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 512, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\mixins.py", line 43, in list
    return self.get_paginated_response(serializer.data)
                                       ^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 797, in data
    ret = super().data
          ^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 251, in data
    self._data = self.to_representation(self.instance)
                 ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 716, in to_representation
    self.child.to_representation(item) for item in iterable
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 540, in to_representation
    ret[field.field_name] = field.to_representation(attribute)
                            ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\fields.py", line 1870, in to_representation
    return method(value)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\properties\serializers.py", line 123, in get_is_favorited
    return obj.favorites.filter(user=request.user).exists()
           ^^^^^^^^^^^^^
AttributeError: 'Property' object has no attribute 'favorites'. Did you mean: 'favorited_by'?
ERROR 2025-10-18 19:06:30,378 basehttp 25156 23152 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 500 172073
INFO 2025-10-18 19:06:39,068 basehttp 25156 23152 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 19:06:39,083 basehttp 25156 23152 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 19:06:39,857 basehttp 25156 10464 "GET /api/auth/me/ HTTP/1.1" 200 887
ERROR 2025-10-18 19:06:39,973 log 25156 23152 Internal Server Error: /api/properties/
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 515, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 475, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 486, in raise_uncaught_exception
    raise exc
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 512, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\mixins.py", line 43, in list
    return self.get_paginated_response(serializer.data)
                                       ^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 797, in data
    ret = super().data
          ^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 251, in data
    self._data = self.to_representation(self.instance)
                 ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 716, in to_representation
    self.child.to_representation(item) for item in iterable
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\serializers.py", line 540, in to_representation
    ret[field.field_name] = field.to_representation(attribute)
                            ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\fields.py", line 1870, in to_representation
    return method(value)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\properties\serializers.py", line 123, in get_is_favorited
    return obj.favorites.filter(user=request.user).exists()
           ^^^^^^^^^^^^^
AttributeError: 'Property' object has no attribute 'favorites'. Did you mean: 'favorited_by'?
ERROR 2025-10-18 19:06:40,006 basehttp 25156 23152 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 500 172073
INFO 2025-10-18 19:09:23,780 autoreload 25156 2136 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\properties\serializers.py changed, reloading.
INFO 2025-10-18 19:09:25,942 autoreload 16504 17888 Watching for file changes with StatReloader
INFO 2025-10-18 19:10:06,152 autoreload 16504 17888 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\properties\serializers.py changed, reloading.
INFO 2025-10-18 19:10:07,635 autoreload 22120 15236 Watching for file changes with StatReloader
INFO 2025-10-18 19:10:18,753 basehttp 22120 3600 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 19:10:18,768 basehttp 22120 3600 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 19:10:19,092 basehttp 22120 22696 "OPTIONS /api/properties/?featured=true&limit=6 HTTP/1.1" 200 0
INFO 2025-10-18 19:10:19,098 basehttp 22120 22696 "OPTIONS /api/auth/me/ HTTP/1.1" 200 0
INFO 2025-10-18 19:10:19,383 basehttp 22120 3600 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-18 19:10:19,385 basehttp 22120 16268 "GET /api/auth/me/ HTTP/1.1" 200 887
INFO 2025-10-18 19:10:19,481 basehttp 22120 16268 "GET /media/property_images/2025/10/18/images_5.jpeg HTTP/1.1" 304 0
INFO 2025-10-18 19:22:38,757 basehttp 22120 19072 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 19:22:39,082 basehttp 22120 7084 "OPTIONS /api/properties/?property_type=rental&status=published HTTP/1.1" 200 0
INFO 2025-10-18 19:22:39,345 basehttp 22120 19072 "GET /api/properties/?property_type=rental&status=published HTTP/1.1" 200 1091
INFO 2025-10-18 19:23:43,282 basehttp 22120 9404 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 19:23:43,500 basehttp 22120 9404 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-18 19:23:43,549 basehttp 22120 9404 "GET /media/property_images/2025/10/18/images_5.jpeg HTTP/1.1" 304 0
INFO 2025-10-18 19:24:15,054 basehttp 22120 23856 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 19:24:15,374 basehttp 22120 18052 "OPTIONS /api/properties/5/ HTTP/1.1" 200 0
INFO 2025-10-18 19:24:15,548 basehttp 22120 23856 "GET /api/properties/5/ HTTP/1.1" 200 4353
INFO 2025-10-18 19:24:15,569 basehttp 22120 23856 "GET /media/property_images/2025/10/18/download_4.jpeg HTTP/1.1" 304 0
INFO 2025-10-18 19:29:45,843 basehttp 22120 16808 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 19:29:46,091 basehttp 22120 16808 "GET /api/properties/? HTTP/1.1" 200 6101
INFO 2025-10-18 19:30:56,700 basehttp 22120 16808 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 19:30:56,886 basehttp 22120 16808 "GET /api/properties/5/ HTTP/1.1" 200 4353
ERROR 2025-10-18 19:51:57,107 log 22120 9956 Internal Server Error: /favicon.ico
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\loader.py", line 47, in select_template
    raise TemplateDoesNotExist(", ".join(template_name_list), chain=chain)
django.template.exceptions.TemplateDoesNotExist: index.html
ERROR 2025-10-18 19:51:57,136 basehttp 22120 9956 "GET /favicon.ico HTTP/1.1" 500 92473
INFO 2025-10-18 19:52:00,265 basehttp 22120 9956 "GET /admin/properties/property/5/change/ HTTP/1.1" 200 116494
ERROR 2025-10-18 19:52:01,044 log 22120 9956 Internal Server Error: /favicon.ico
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\loader.py", line 47, in select_template
    raise TemplateDoesNotExist(", ".join(template_name_list), chain=chain)
django.template.exceptions.TemplateDoesNotExist: index.html
ERROR 2025-10-18 19:52:01,056 basehttp 22120 9956 "GET /favicon.ico HTTP/1.1" 500 92508
INFO 2025-10-18 19:52:23,881 basehttp 22120 9956 "GET /admin/properties/propertyimage/add/ HTTP/1.1" 200 22459
INFO 2025-10-18 19:52:24,021 basehttp 22120 9956 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
ERROR 2025-10-18 19:52:24,267 log 22120 9956 Internal Server Error: /favicon.ico
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\loader.py", line 47, in select_template
    raise TemplateDoesNotExist(", ".join(template_name_list), chain=chain)
django.template.exceptions.TemplateDoesNotExist: index.html
ERROR 2025-10-18 19:52:24,275 basehttp 22120 9956 "GET /favicon.ico HTTP/1.1" 500 92508
INFO 2025-10-18 19:52:29,330 basehttp 22120 9956 "GET /admin/properties/propertymedia/add/ HTTP/1.1" 200 24817
INFO 2025-10-18 19:52:29,607 basehttp 22120 9956 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
ERROR 2025-10-18 19:52:29,805 log 22120 9956 Internal Server Error: /favicon.ico
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\loader.py", line 47, in select_template
    raise TemplateDoesNotExist(", ".join(template_name_list), chain=chain)
django.template.exceptions.TemplateDoesNotExist: index.html
ERROR 2025-10-18 19:52:29,813 basehttp 22120 9956 "GET /favicon.ico HTTP/1.1" 500 92508
INFO 2025-10-18 19:54:06,484 basehttp 22120 2252 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 19:54:07,126 basehttp 22120 2252 "GET /api/properties/5/ HTTP/1.1" 200 4353
INFO 2025-10-18 19:54:07,180 basehttp 22120 2252 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 19:54:07,499 basehttp 22120 19460 "OPTIONS /api/properties/5/increment-views/ HTTP/1.1" 200 0
INFO 2025-10-18 19:54:07,680 basehttp 22120 2252 "GET /api/properties/5/increment-views/ HTTP/1.1" 200 17
INFO 2025-10-18 19:54:35,263 basehttp 22120 16552 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 19:54:35,487 basehttp 22120 16552 "GET /api/properties/5/ HTTP/1.1" 200 4353
INFO 2025-10-18 19:54:35,497 basehttp 22120 16552 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 19:54:35,654 basehttp 22120 16552 "GET /api/properties/5/increment-views/ HTTP/1.1" 200 17
INFO 2025-10-18 19:55:02,993 basehttp 22120 4916 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 19:55:03,283 basehttp 22120 4916 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-18 19:55:03,335 basehttp 22120 4916 "GET /media/property_images/2025/10/18/images_5.jpeg HTTP/1.1" 304 0
INFO 2025-10-18 19:55:21,601 basehttp 22120 16140 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 19:55:21,842 basehttp 22120 16140 "GET /api/properties/5/ HTTP/1.1" 200 4353
INFO 2025-10-18 19:55:21,871 basehttp 22120 16140 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 19:55:22,006 basehttp 22120 16140 "GET /api/properties/5/increment-views/ HTTP/1.1" 200 17
INFO 2025-10-18 19:55:22,032 basehttp 22120 16140 "GET /media/property_images/2025/10/18/download_4.jpeg HTTP/1.1" 304 0
INFO 2025-10-18 19:58:39,386 basehttp 22120 21808 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 19:58:39,640 basehttp 22120 21808 "GET /api/properties/? HTTP/1.1" 200 6101
INFO 2025-10-18 19:58:41,326 basehttp 22120 21808 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 19:58:41,598 basehttp 22120 21808 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-18 20:13:32,076 basehttp 22120 2740 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-18 20:13:32,489 basehttp 22120 2740 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-18 20:13:32,556 basehttp 22120 2740 "GET /media/property_images/2025/10/18/images_5.jpeg HTTP/1.1" 304 0
INFO 2025-10-18 20:38:38,932 autoreload 22120 15236 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\pristineprimer\settings.py changed, reloading.
INFO 2025-10-18 20:38:42,427 autoreload 15156 21852 Watching for file changes with StatReloader
INFO 2025-10-18 20:58:24,059 autoreload 15156 21852 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\pristineprimer\settings.py changed, reloading.
INFO 2025-10-19 23:50:35,614 autoreload 25720 25856 Watching for file changes with StatReloader
INFO 2025-10-19 23:50:44,886 basehttp 25720 17192 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-19 23:50:44,891 basehttp 25720 17192 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-19 23:50:45,205 basehttp 25720 11656 "OPTIONS /api/auth/me/ HTTP/1.1" 200 0
INFO 2025-10-19 23:50:45,206 basehttp 25720 26352 "OPTIONS /api/properties/?featured=true&limit=6 HTTP/1.1" 200 0
INFO 2025-10-19 23:50:47,439 basehttp 25720 23156 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-19 23:50:47,443 basehttp 25720 23156 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-19 23:50:47,763 basehttp 25720 7084 "OPTIONS /api/properties/?featured=true&limit=6 HTTP/1.1" 200 0
INFO 2025-10-19 23:50:47,765 basehttp 25720 2276 "OPTIONS /api/auth/me/ HTTP/1.1" 200 0
INFO 2025-10-19 23:50:49,674 basehttp 25720 23156 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-19 23:50:49,696 basehttp 25720 23156 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-19 23:50:49,723 basehttp 25720 2276 "OPTIONS /api/properties/?featured=true&limit=6 HTTP/1.1" 200 0
INFO 2025-10-19 23:50:49,759 basehttp 25720 2276 "OPTIONS /api/auth/me/ HTTP/1.1" 200 0
INFO 2025-10-19 23:51:31,806 autoreload 25720 25856 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\pristineprimer\settings.py changed, reloading.
INFO 2025-10-19 23:51:33,300 autoreload 26592 22084 Watching for file changes with StatReloader
INFO 2025-10-19 23:51:49,070 autoreload 26592 22084 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\pristineprimer\settings.py changed, reloading.
INFO 2025-10-19 23:51:51,718 autoreload 16620 4164 Watching for file changes with StatReloader
INFO 2025-10-19 23:52:02,766 autoreload 16620 4164 C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\pristineprimer\settings.py changed, reloading.
INFO 2025-10-19 23:52:04,202 autoreload 4348 19792 Watching for file changes with StatReloader
INFO 2025-10-19 23:52:13,882 basehttp 4348 27380 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-19 23:52:13,883 basehttp 4348 22320 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-19 23:52:14,207 basehttp 4348 20844 "OPTIONS /api/auth/me/ HTTP/1.1" 200 0
INFO 2025-10-19 23:52:14,207 basehttp 4348 26632 "OPTIONS /api/properties/?featured=true&limit=6 HTTP/1.1" 200 0
INFO 2025-10-19 23:52:14,402 basehttp 4348 22320 "GET /api/auth/me/ HTTP/1.1" 200 887
INFO 2025-10-19 23:52:14,443 basehttp 4348 27380 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-19 23:52:14,512 basehttp 4348 27380 "GET /media/property_images/2025/10/18/images_5.jpeg HTTP/1.1" 304 0
INFO 2025-10-19 23:52:14,512 basehttp 4348 22320 "GET /media/property_images/ChatGPT_Image_Sep_13_2025_02_47_02_PM.png HTTP/1.1" 304 0
INFO 2025-10-19 23:52:14,513 basehttp 4348 10004 "GET /media/property_images/ChatGPT_Image_Sep_13_2025_02_47_02_PM_obfB0IH.png HTTP/1.1" 304 0
INFO 2025-10-19 23:53:05,191 basehttp 4348 15964 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-19 23:53:05,906 basehttp 4348 15964 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-19 23:53:27,097 basehttp 4348 18064 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-19 23:53:27,510 basehttp 4348 18064 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-19 23:55:45,034 basehttp 4348 11120 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-19 23:55:45,289 basehttp 4348 11120 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-19 23:56:06,127 basehttp 4348 23028 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-19 23:56:06,335 basehttp 4348 23028 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-19 23:56:09,694 basehttp 4348 22316 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-19 23:56:10,020 basehttp 4348 25336 "OPTIONS /api/properties/2/ HTTP/1.1" 200 0
INFO 2025-10-19 23:56:10,181 basehttp 4348 22316 "GET /api/properties/2/ HTTP/1.1" 200 1809
INFO 2025-10-19 23:56:10,187 basehttp 4348 22316 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-19 23:56:10,196 basehttp 4348 25336 "OPTIONS /api/properties/2/increment-views/ HTTP/1.1" 200 0
INFO 2025-10-19 23:56:10,316 basehttp 4348 22316 "GET /api/properties/2/increment-views/ HTTP/1.1" 200 18
INFO 2025-10-19 23:56:14,050 basehttp 4348 22316 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-19 23:56:14,366 basehttp 4348 22316 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-20 00:06:39,394 basehttp 4348 9516 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:06:39,639 basehttp 4348 9516 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-20 00:08:47,931 basehttp 4348 9516 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:08:48,141 basehttp 4348 9516 "GET /api/properties/2/ HTTP/1.1" 200 1809
INFO 2025-10-20 00:08:48,149 basehttp 4348 9516 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:08:48,274 basehttp 4348 9516 "GET /api/properties/2/increment-views/ HTTP/1.1" 200 18
INFO 2025-10-20 00:08:49,936 basehttp 4348 9516 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:08:50,258 basehttp 4348 16824 "OPTIONS /api/properties/? HTTP/1.1" 200 0
INFO 2025-10-20 00:08:50,491 basehttp 4348 9516 "GET /api/properties/? HTTP/1.1" 200 6101
INFO 2025-10-20 00:08:51,920 basehttp 4348 9516 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:08:52,140 basehttp 4348 9516 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-20 00:09:17,454 basehttp 4348 9516 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:09:17,901 basehttp 4348 9516 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-20 00:09:40,767 basehttp 4348 9516 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:09:41,108 basehttp 4348 9516 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-20 00:10:00,729 basehttp 4348 9516 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:10:01,040 basehttp 4348 9516 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-20 00:11:15,983 basehttp 4348 9516 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:11:17,617 basehttp 4348 9516 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-20 00:11:31,823 basehttp 4348 9516 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:11:32,128 basehttp 4348 9516 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-20 00:12:41,073 basehttp 4348 26468 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:12:41,480 basehttp 4348 26468 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-20 00:12:46,713 basehttp 4348 26468 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:12:46,983 basehttp 4348 26468 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-20 00:12:52,745 basehttp 4348 26468 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:12:53,031 basehttp 4348 26468 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-20 00:13:09,032 basehttp 4348 4216 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:13:09,304 basehttp 4348 4216 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-20 00:14:00,915 basehttp 4348 4216 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:14:01,475 basehttp 4348 4216 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-20 00:14:15,757 basehttp 4348 4216 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:14:16,175 basehttp 4348 4216 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-20 00:14:55,092 basehttp 4348 13720 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:14:55,294 basehttp 4348 13720 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-20 00:15:32,739 basehttp 4348 13720 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:15:32,924 basehttp 4348 13720 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-20 00:15:44,062 basehttp 4348 10916 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:15:44,234 basehttp 4348 10916 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-20 00:15:55,070 basehttp 4348 26356 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:15:55,844 basehttp 4348 26356 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-20 00:16:10,049 basehttp 4348 17232 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:16:10,305 basehttp 4348 17232 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-20 00:16:34,636 basehttp 4348 25976 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:16:35,037 basehttp 4348 25976 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-20 00:17:36,052 basehttp 4348 9452 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:17:36,411 basehttp 4348 9452 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-20 00:20:45,135 basehttp 4348 8204 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:20:45,306 basehttp 4348 8204 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-20 00:21:03,643 basehttp 4348 26400 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:21:04,214 basehttp 4348 26400 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-20 00:21:15,049 basehttp 4348 12412 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:21:15,304 basehttp 4348 12412 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-20 00:21:57,057 basehttp 4348 3720 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:21:57,304 basehttp 4348 3720 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-20 00:32:13,918 basehttp 4348 20784 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:32:14,244 basehttp 4348 27044 "OPTIONS /api/properties/5/ HTTP/1.1" 200 0
INFO 2025-10-20 00:32:14,587 basehttp 4348 20784 "GET /api/properties/5/ HTTP/1.1" 200 4353
INFO 2025-10-20 00:32:14,595 basehttp 4348 20784 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:32:14,602 basehttp 4348 27044 "OPTIONS /api/properties/5/increment-views/ HTTP/1.1" 200 0
INFO 2025-10-20 00:32:14,726 basehttp 4348 20784 "GET /api/properties/5/increment-views/ HTTP/1.1" 200 17
INFO 2025-10-20 00:32:14,766 basehttp 4348 20784 "GET /media/property_images/2025/10/18/download_4.jpeg HTTP/1.1" 304 0
INFO 2025-10-20 00:32:16,605 basehttp 4348 20784 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:32:16,799 basehttp 4348 20784 "GET /api/properties/? HTTP/1.1" 200 6101
INFO 2025-10-20 00:32:26,519 basehttp 4348 15940 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:32:26,723 basehttp 4348 15940 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-20 00:40:28,706 basehttp 4348 24068 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:40:28,996 basehttp 4348 24068 "GET /api/properties/5/ HTTP/1.1" 200 4353
INFO 2025-10-20 00:40:29,009 basehttp 4348 24068 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:40:29,140 basehttp 4348 24068 "GET /api/properties/5/increment-views/ HTTP/1.1" 200 17
INFO 2025-10-20 00:40:40,640 basehttp 4348 25156 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:40:40,796 basehttp 4348 25156 "GET /api/properties/? HTTP/1.1" 200 6101
INFO 2025-10-20 00:51:39,993 basehttp 4348 11808 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:51:40,424 basehttp 4348 11808 "GET /api/properties/? HTTP/1.1" 200 6101
INFO 2025-10-20 00:51:44,678 basehttp 4348 25952 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:51:44,861 basehttp 4348 25952 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-20 00:51:58,401 basehttp 4348 21668 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:51:58,714 basehttp 4348 1888 "OPTIONS /api/properties/?property_type=rental&status=published HTTP/1.1" 200 0
INFO 2025-10-20 00:51:58,899 basehttp 4348 21668 "GET /api/properties/?property_type=rental&status=published HTTP/1.1" 200 1091
INFO 2025-10-20 00:54:49,082 basehttp 4348 22660 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:54:49,707 basehttp 4348 22660 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-20 00:57:01,557 basehttp 4348 16620 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:57:01,700 basehttp 4348 16620 "GET /api/properties/?property_type=rental&status=published HTTP/1.1" 200 1091
INFO 2025-10-20 00:57:08,556 basehttp 4348 16620 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:57:08,773 basehttp 4348 16620 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-20 00:57:17,826 basehttp 4348 10984 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:57:17,997 basehttp 4348 10984 "GET /api/properties/? HTTP/1.1" 200 6101
INFO 2025-10-20 00:57:23,929 basehttp 4348 2264 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 00:57:24,063 basehttp 4348 2264 "GET /api/properties/?property_type=rental&status=published HTTP/1.1" 200 1091
INFO 2025-10-20 01:00:13,286 basehttp 4348 24368 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 01:00:13,468 basehttp 4348 24368 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-20 01:01:03,024 basehttp 4348 25956 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 01:01:03,194 basehttp 4348 25956 "GET /api/properties/?property_type=rental&status=published HTTP/1.1" 200 1091
INFO 2025-10-20 01:01:11,470 basehttp 4348 25956 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 01:01:11,758 basehttp 4348 25956 "GET /api/properties/? HTTP/1.1" 200 6101
INFO 2025-10-20 01:01:16,914 basehttp 4348 25956 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 01:01:17,260 basehttp 4348 25408 "OPTIONS /api/properties/?property_type=land HTTP/1.1" 200 0
INFO 2025-10-20 01:01:17,426 basehttp 4348 25956 "GET /api/properties/?property_type=land HTTP/1.1" 200 2032
INFO 2025-10-20 01:01:40,450 basehttp 4348 22308 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 01:01:40,603 basehttp 4348 22308 "GET /api/properties/?property_type=rental&status=published HTTP/1.1" 200 1091
INFO 2025-10-20 01:12:53,854 basehttp 4348 18092 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 01:12:54,290 basehttp 4348 18092 "GET /api/properties/?property_type=rental&status=published HTTP/1.1" 200 1091
INFO 2025-10-20 01:32:14,314 basehttp 4348 12464 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 01:32:14,769 basehttp 4348 12464 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-20 01:59:05,118 basehttp 4348 5980 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 01:59:05,436 basehttp 4348 25076 "OPTIONS /api/properties/?featured=true&limit=6 HTTP/1.1" 200 0
INFO 2025-10-20 01:59:07,162 basehttp 4348 19292 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-20 01:59:21,197 basehttp 4348 12420 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 01:59:21,205 basehttp 4348 12420 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 01:59:21,528 basehttp 4348 2476 "OPTIONS /api/auth/dashboard/overview/ HTTP/1.1" 200 0
INFO 2025-10-20 01:59:21,528 basehttp 4348 23748 "OPTIONS /api/properties/my_properties/ HTTP/1.1" 200 0
ERROR 2025-10-20 01:59:22,009 log 4348 3160 Internal Server Error: /api/auth/dashboard/overview/
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\views\generic\base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 515, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 475, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 486, in raise_uncaught_exception
    raise exc
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 512, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\decorators.py", line 50, in handler
    return func(*args, **kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\users\views.py", line 183, in dashboard_overview
    my_listings = Property.objects.filter(owner=user)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\query.py", line 1493, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\query.py", line 1511, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\query.py", line 1518, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
    ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\sql\query.py", line 1646, in add_q
    clause, _ = self._add_q(q_object, can_reuse)
                ~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\sql\query.py", line 1678, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ~~~~~~~~~~~~~~~~~^
        child,
        ^^^^^^
    ...<7 lines>...
        update_join_types=update_join_types,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\sql\query.py", line 1526, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg, summarize)
                                        ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\sql\query.py", line 1333, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
                                ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\sql\query.py", line 1806, in names_to_path
    raise FieldError(
    ...<2 lines>...
    )
django.core.exceptions.FieldError: Cannot resolve keyword 'owner' into field. Choices are: address, agent, agent_id, amenities, bathrooms, bedrooms, city, contact_info, created_at, deposit_percentage, description, discount_offers, distance_to_main_road, documents, electricity_availability, favorited_by, featured, has_beacons, has_borehole, has_central_air, has_drainage, has_fireplace, has_garage, has_garden, has_piped_water, has_pool, has_sewer_system, has_subdivision_approval, id, images, inquiries, inquiry_count, internet_availability, is_fenced, is_gated_community, is_negotiable, land_type, landmarks, latitude, longitude, lot_size, media, num_plots_available, payment_plan_details, plot_dimensions, price, price_per_unit, price_unit, property_type, published_at, road_access_type, seller, seller_id, short_description, size_acres, soil_type, square_feet, state, status, title, title_deed_status, topography, total_plots, updated_at, useractivity, views_count, water_supply_types, year_built, zip_code, zoning
ERROR 2025-10-20 01:59:22,064 basehttp 4348 3160 "GET /api/auth/dashboard/overview/ HTTP/1.1" 500 164367
INFO 2025-10-20 01:59:22,077 basehttp 4348 2624 "GET /api/properties/my_properties/ HTTP/1.1" 200 3958
INFO 2025-10-20 01:59:27,958 basehttp 4348 2624 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 01:59:28,192 basehttp 4348 2624 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-20 02:07:10,861 basehttp 4348 4256 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 02:07:11,135 basehttp 4348 4256 "GET /api/properties/? HTTP/1.1" 200 6101
INFO 2025-10-20 02:07:15,773 basehttp 4348 4256 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 02:07:15,925 basehttp 4348 4256 "GET /api/properties/?property_type=rental&status=published HTTP/1.1" 200 1091
INFO 2025-10-20 02:07:25,826 basehttp 4348 22308 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 02:07:25,837 basehttp 4348 22308 "GET /api/auth/csrf/ HTTP/1.1" 200 81
ERROR 2025-10-20 02:07:26,039 log 4348 22308 Internal Server Error: /api/auth/dashboard/overview/
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\views\generic\base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 515, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 475, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 486, in raise_uncaught_exception
    raise exc
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 512, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\decorators.py", line 50, in handler
    return func(*args, **kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\users\views.py", line 183, in dashboard_overview
    my_listings = Property.objects.filter(owner=user)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\query.py", line 1493, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\query.py", line 1511, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\query.py", line 1518, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
    ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\sql\query.py", line 1646, in add_q
    clause, _ = self._add_q(q_object, can_reuse)
                ~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\sql\query.py", line 1678, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ~~~~~~~~~~~~~~~~~^
        child,
        ^^^^^^
    ...<7 lines>...
        update_join_types=update_join_types,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\sql\query.py", line 1526, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg, summarize)
                                        ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\sql\query.py", line 1333, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
                                ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\sql\query.py", line 1806, in names_to_path
    raise FieldError(
    ...<2 lines>...
    )
django.core.exceptions.FieldError: Cannot resolve keyword 'owner' into field. Choices are: address, agent, agent_id, amenities, bathrooms, bedrooms, city, contact_info, created_at, deposit_percentage, description, discount_offers, distance_to_main_road, documents, electricity_availability, favorited_by, featured, has_beacons, has_borehole, has_central_air, has_drainage, has_fireplace, has_garage, has_garden, has_piped_water, has_pool, has_sewer_system, has_subdivision_approval, id, images, inquiries, inquiry_count, internet_availability, is_fenced, is_gated_community, is_negotiable, land_type, landmarks, latitude, longitude, lot_size, media, num_plots_available, payment_plan_details, plot_dimensions, price, price_per_unit, price_unit, property_type, published_at, road_access_type, seller, seller_id, short_description, size_acres, soil_type, square_feet, state, status, title, title_deed_status, topography, total_plots, updated_at, useractivity, views_count, water_supply_types, year_built, zip_code, zoning
ERROR 2025-10-20 02:07:26,056 basehttp 4348 22308 "GET /api/auth/dashboard/overview/ HTTP/1.1" 500 164367
INFO 2025-10-20 02:07:26,248 basehttp 4348 27216 "GET /api/properties/my_properties/ HTTP/1.1" 200 3958
INFO 2025-10-20 02:08:02,114 basehttp 4348 10996 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 02:08:02,291 basehttp 4348 10996 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-20 02:08:18,527 basehttp 4348 10996 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 02:08:18,673 basehttp 4348 10996 "GET /api/properties/? HTTP/1.1" 200 6101
INFO 2025-10-20 02:08:21,494 basehttp 4348 10996 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 02:08:21,671 basehttp 4348 10996 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-20 02:08:53,621 basehttp 4348 10996 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 02:08:53,841 basehttp 4348 10996 "GET /api/properties/5/ HTTP/1.1" 200 4353
INFO 2025-10-20 02:08:53,850 basehttp 4348 10996 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 02:08:53,963 basehttp 4348 10996 "GET /api/properties/5/increment-views/ HTTP/1.1" 200 17
INFO 2025-10-20 02:08:59,720 basehttp 4348 10996 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 02:08:59,909 basehttp 4348 10996 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-20 02:09:05,666 basehttp 4348 10996 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 02:09:05,677 basehttp 4348 10996 "GET /api/auth/csrf/ HTTP/1.1" 200 81
ERROR 2025-10-20 02:09:06,061 log 4348 10996 Internal Server Error: /api/auth/dashboard/overview/
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\views\generic\base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 515, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 475, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 486, in raise_uncaught_exception
    raise exc
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\views.py", line 512, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\rest_framework\decorators.py", line 50, in handler
    return func(*args, **kwargs)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\users\views.py", line 183, in dashboard_overview
    my_listings = Property.objects.filter(owner=user)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\query.py", line 1493, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\query.py", line 1511, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\query.py", line 1518, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
    ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\sql\query.py", line 1646, in add_q
    clause, _ = self._add_q(q_object, can_reuse)
                ~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\sql\query.py", line 1678, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ~~~~~~~~~~~~~~~~~^
        child,
        ^^^^^^
    ...<7 lines>...
        update_join_types=update_join_types,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\sql\query.py", line 1526, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg, summarize)
                                        ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\sql\query.py", line 1333, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
                                ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\db\models\sql\query.py", line 1806, in names_to_path
    raise FieldError(
    ...<2 lines>...
    )
django.core.exceptions.FieldError: Cannot resolve keyword 'owner' into field. Choices are: address, agent, agent_id, amenities, bathrooms, bedrooms, city, contact_info, created_at, deposit_percentage, description, discount_offers, distance_to_main_road, documents, electricity_availability, favorited_by, featured, has_beacons, has_borehole, has_central_air, has_drainage, has_fireplace, has_garage, has_garden, has_piped_water, has_pool, has_sewer_system, has_subdivision_approval, id, images, inquiries, inquiry_count, internet_availability, is_fenced, is_gated_community, is_negotiable, land_type, landmarks, latitude, longitude, lot_size, media, num_plots_available, payment_plan_details, plot_dimensions, price, price_per_unit, price_unit, property_type, published_at, road_access_type, seller, seller_id, short_description, size_acres, soil_type, square_feet, state, status, title, title_deed_status, topography, total_plots, updated_at, useractivity, views_count, water_supply_types, year_built, zip_code, zoning
ERROR 2025-10-20 02:09:06,252 basehttp 4348 10996 "GET /api/auth/dashboard/overview/ HTTP/1.1" 500 164367
INFO 2025-10-20 02:09:06,421 basehttp 4348 17832 "GET /api/properties/my_properties/ HTTP/1.1" 200 3958
INFO 2025-10-20 02:26:44,369 basehttp 4348 25140 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 02:26:44,680 basehttp 4348 25140 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-20 02:26:56,809 basehttp 4348 25140 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 02:26:57,184 basehttp 4348 4544 "OPTIONS /api/properties/? HTTP/1.1" 200 0
INFO 2025-10-20 02:26:57,706 basehttp 4348 25140 "GET /api/properties/? HTTP/1.1" 200 6101
INFO 2025-10-20 02:26:58,721 basehttp 4348 25140 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 02:26:59,152 basehttp 4348 25140 "GET /api/properties/?property_type=rental&status=published HTTP/1.1" 200 1091
INFO 2025-10-20 02:27:14,553 basehttp 4348 25140 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 02:27:14,971 basehttp 4348 25140 "GET /api/properties/? HTTP/1.1" 200 6101
INFO 2025-10-20 02:27:17,996 basehttp 4348 25140 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 02:27:18,386 basehttp 4348 25140 "GET /api/properties/?featured=true&limit=6 HTTP/1.1" 200 4125
INFO 2025-10-20 02:37:25,154 basehttp 4348 23484 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 02:37:25,675 basehttp 4348 23484 "GET /api/properties/? HTTP/1.1" 200 6101
INFO 2025-10-20 02:37:28,532 basehttp 4348 23484 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 02:37:28,828 basehttp 4348 23484 "GET /api/properties/?property_type=rental&status=published HTTP/1.1" 200 1091
INFO 2025-10-20 13:43:20,269 basehttp 4348 3496 "OPTIONS /api/properties/?property_type=rental&status=published HTTP/1.1" 200 0
INFO 2025-10-20 13:43:20,272 basehttp 4348 25612 "OPTIONS /api/auth/me/ HTTP/1.1" 200 0
INFO 2025-10-20 13:43:21,365 basehttp 4348 21304 "GET /api/auth/me/ HTTP/1.1" 200 887
INFO 2025-10-20 13:43:21,407 basehttp 4348 27732 "GET /api/properties/?property_type=rental&status=published HTTP/1.1" 200 1091
INFO 2025-10-20 16:22:51,782 basehttp 4348 27292 "OPTIONS /api/properties/?property_type=rental&status=published HTTP/1.1" 200 0
INFO 2025-10-20 16:22:51,791 basehttp 4348 1776 "OPTIONS /api/auth/me/ HTTP/1.1" 200 0
INFO 2025-10-20 16:22:52,769 basehttp 4348 24680 "GET /api/auth/me/ HTTP/1.1" 200 887
INFO 2025-10-20 16:22:52,829 basehttp 4348 20848 "GET /api/properties/?property_type=rental&status=published HTTP/1.1" 200 1091
INFO 2025-10-20 16:22:55,269 basehttp 4348 20848 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 16:22:55,274 basehttp 4348 1776 "OPTIONS /api/properties/1/ HTTP/1.1" 200 0
INFO 2025-10-20 16:22:55,476 basehttp 4348 20848 "GET /api/properties/1/ HTTP/1.1" 200 2055
INFO 2025-10-20 16:22:55,486 basehttp 4348 20848 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 16:22:55,493 basehttp 4348 1776 "OPTIONS /api/properties/1/increment-views/ HTTP/1.1" 200 0
INFO 2025-10-20 16:22:55,666 basehttp 4348 20848 "GET /api/properties/1/increment-views/ HTTP/1.1" 200 18
INFO 2025-10-20 16:22:55,895 basehttp 4348 20848 "GET /media/property_images/delivery-man-on-motorcycle-with-package-vector.jpg HTTP/1.1" 200 72060
INFO 2025-10-20 16:23:17,027 basehttp 4348 9060 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 16:23:17,236 basehttp 4348 9060 "GET /api/properties/1/ HTTP/1.1" 200 2055
INFO 2025-10-20 16:23:17,247 basehttp 4348 9060 "GET /api/auth/csrf/ HTTP/1.1" 200 81
INFO 2025-10-20 16:23:17,372 basehttp 4348 9060 "GET /api/properties/1/increment-views/ HTTP/1.1" 200 18
WARNING 2025-10-20 22:15:58,425 log 4348 26684 Not Found: /api/orders/cart/
WARNING 2025-10-20 22:15:58,448 basehttp 4348 26684 "GET /api/orders/cart/ HTTP/1.1" 404 30377
ERROR 2025-10-20 22:15:59,083 log 4348 26684 Internal Server Error: /favicon.ico
Traceback (most recent call last):
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "C:\Users\User1\Documents\PayedProjects\Pristineprimier_real_estate_Website\server\venv\Lib\site-packages\django\template\loader.py", line 47, in select_template
    raise TemplateDoesNotExist(", ".join(template_name_list), chain=chain)
django.template.exceptions.TemplateDoesNotExist: index.html
ERROR 2025-10-20 22:15:59,126 basehttp 4348 26684 "GET /favicon.ico HTTP/1.1" 500 92301
